data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/SearchConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SearchConfiguration {
+
+    @JsonProperty("batch_size")
+    private Integer batchSize;
+
+    @JsonProperty("expand_wildcards")
+    private String expandWildcards;
```

Comment:
This should probably be an enum with these options `[ "open", "closed", "none", "all"]`, or at least add the validations for these like my comment on the `order` parameter for sorting.

This should default to `all`

Reply:
Thanks @graytaylor0  for the review comments. The changes are incorporated.